### DISCUSSION TITLE
Support for google, facebook and microsoft authentication with configuration.

### DISCUSSION THREAD

**Discussion by ShankarlalChaurasiya** [NeoSOFT-Technologies/rest-dot-net-core#76] [Ideas]
Nice to have  google, facebook and microsoft  login authentication, two factor authentication with microsoft, google authenticator, otp from sms or application with configuration as per requirement.

**Comment by sasachan** [comment]
Thank you @ShankarlalChaurasiya 

We have accepted your feature request and this would added to [R2](https://github.com/NeoSOFT-Technologies/rest-dot-net-core/milestone/2) as future milestone.
